packages/@blockprotocol/graph/package.json

```diff
@@ -21,16 +21,35 @@
     "name": "HASH",
     "url": "https://hash.ai"
   },
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.js",
-  "types": "./dist/esm/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js",
+    "./react": "./dist/react.js",
+    "./custom-element": "./dist/custom-element.js",
+    "./graph-service-json": "./dist/graph-service-json.js"
```

Comment:
What are the downsides of re-exporting this payload from `index.ts`?

Reply:
From memory it caused an issue when trying to import it into a package which didn't need the JSON but had to evaluate the import statement. I can't remember exactly what – probably using `readFileSync` instead of import will eliminate whatever the problem was so I will do that at the same time. [task](https://app.asana.com/0/1202538466812818/1202614421149286/f)